v2g-liberty/.vscode/tasks.json

```diff
@@ -9,11 +9,23 @@
       "command": "python -m appdaemon -c rootfs/root/appdaemon -C appdaemon.devcontainer.yaml -D INFO",
       "problemMatcher": []
     },
+    {
```

Comment:
Sometimes Ctrl-C does not seem to work, so I added an extra task to kill the running V2G-Librty app in the devcontainer.